website/docs/registry/providers/docs.mdx

```diff
@@ -9,6 +9,8 @@ description: >-
 
 The [Terraform Registry][terraform-registry] displays documentation for the providers it hosts. This page describes the expected format for provider documentation.
```

Comment:
```suggestion
The [Terraform Registry][terraform-registry] displays documentation for the providers it hosts. This page describes the expected format for provider documentation. Provider documentation is _not_ supported in the private registry of [Terraform Cloud or Terraform Enterprise](/terraform/cloud-docs/registry/publish-providers#release-files).
```

Reply:
I suggested moving this sentence up, adding emphasis to _not_, remove the "isn't" contractions (for style we avoid contraction in the docs), and shortened the link above. Please let me know if any of those don't feel good!

Reply:
I'm perfectly alright with this change. Thank you for helping